support-frontend/assets/pages/contributions-landing/components/StripePaymentRequestButton/StripePaymentRequestButton.tsx

```diff
@@ -309,22 +317,25 @@ function onPayment(
 		);
 	// Single doesn't need a state, however recurring (i.e. Zuora) needs a valid state for US and CA billing countries.
 	const validatedCountryFromCard: Option<IsoCountry> = findIsoCountry(
-		billingCountryFromCard,
+		billingCountryFromCard ?? undefined,
```

Comment:
This was a problem I was running into quite a lot, we seem to have a complete mix of functions that return `T | null` or `T | undefined` which meant I was having to map that `null` on to `undefined` quite a lot throughout this file

Reply:
Yeah I've seen we're often passing `null` where maybe we could make a property/argument optional instead